### ISSUE TITLE
Bug in drift metric

### ISSUE THREAD

**Issue by mustaszewski** [mariru/dynamic_bernoulli_embeddings#11]
It seems that there is a minor bug in the `detect_drift` function of `models.py` (https://github.com/mariru/dynamic_bernoulli_embeddings/blob/master/src/models.py).
The code in line 311 is:

`tf_dist, tf_w_idx = tf.nn.top_k(tf.reduce_sum(tf.square(self.rho_t[self.T-1]-self.rho_t[0]),1), 500)`

According to Equation 9 of the paper (https://dl.acm.org/doi/fullHtml/10.1145/3178876.3185999#eq9), drift is calculated using Euclidean distance. However, the code cited above lacks the square root, which is required to calculate Euclidean distance, so the code should rather be something like

`tf_dist, tf_w_idx = tf.nn.top_k(tf.sqrt(tf.reduce_sum(tf.square(self.rho_t[self.T-1]-self.rho_t[0]),1)), 500)
`
or
`f_w_idx = tf.nn.top_k(tf.norm(self.rho_t[self.T-1]-self.rho_t[0], ord = "euclidean"), 500)`
 
Due to the above bug, the drift scores from the second column of the output file top_drifting_words.txt are computer different than Equation 9 suggests.
